### PR TITLE
Init Model

### PR DESCRIPTION
**Pull request by tatuan19**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/40029)

### CODE REVIEW

**Comment by tatuan19** [comment]
ready

**Comment by quan191** [comment]
ok

**Comment by datbk206** [comment]
ok

**Comment by tatuan19** [comment]
ready

File: app/models/payment.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Payment < ApplicationRecord
+  validates :amount, presence: true, numericality: {greater_than: 0}
```
Comment (ngocvt-0484) [inline app/models/payment.rb]: 0 vô setting

File: app/models/booking.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class Booking < ApplicationRecord
+  belongs_to :room
+  belongs_to :user
+  has_one :payment, dependent: :destroy
+
+  validates :room_id, presence: true
```
Comment (ngocvt-0484) [inline app/models/booking.rb]: k cần validate này
mặc định belongs_to :room nó tự sinh ra r

File: app/models/room_type.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class RoomType < ApplicationRecord
+  has_many :rooms, dependent: :destroy
+
+  validates :name, presence: true
+  validates :description, presence: true,
+                  length: {maximum: Settings.validation.description.length.max}
+  validates :bed_num, :cost, presence: true, numericality: {greater_than: 0}
```
Comment (ngocvt-0484) [inline app/models/room_type.rb]: same. add vô setting

**Comment by tatuan19** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**